src/ports/rt-kernel/snmp_mib2.c

```diff
@@ -0,0 +1,137 @@
+/*********************************************************************
```

Comment:
I don't think we can add our own header to this file. BTW, we will update the cmake toolchain so that linker should be able to resolve circular dependencies (by wrapping all libs in -Wl,--start-group/-Wl,--end-group), if that would help.

Reply:
Ok, I'll remove the header.

The mentionened cmake change could make it possible to remove this file.

Reply:
Entire file removed. Similar header was also removed in another file.